### PR TITLE
Create debian package for Mavlink router sample apps

### PR DESCRIPTION
**Pull request by Rjasuja**
Add package for Mavlink router python examples

### CODE REVIEW

**Review by anitha-suresh-zz — COMMENTED** [review]
Commit message added, is explaining about the dependency but not about the commit.
Dependency cannot be mentioned here.

**Comment by Rjasuja** [comment]
To mention, Examples have dependency on pymavlink which can be installed using pip.

File: aero-system/aero-system [hunk 1]
```diff
@@ -6,7 +6,7 @@ Standards-Version: 3.9.2
 Package: aero-system
 Version: 1.0.10
 Maintainer: Daniel E Ruano <daniel.e.ruano@intel.com>
-Depends: linux-image-4.4.76-aero-1.2, linux-headers-4.4.76-aero-1.2, aero-mavlink-router, aero-fpga, aero-px4, aero-ardupilot, aero-utils, aero-init, aero-bios, openssh-server, i2c-tools, firmware-atomisp, aero-systemd, aero-camera-streaming-daemon, aero-optical-flow
+Depends: linux-image-4.4.76-aero-1.2, linux-headers-4.4.76-aero-1.2, aero-mavlink-router, aero-fpga, aero-px4, aero-ardupilot, aero-utils, aero-init, aero-bios, openssh-server, i2c-tools, firmware-atomisp, aero-systemd, aero-camera-streaming-daemon, aero-optical-flow, aero-mavlink-router-samples
```
Comment (avinash-palleti) [inline aero-system/aero-system]: There is another virtual package for aero-samples. Add aero-mavlink-router-samples as dependency there.

File: aero-mavlink-router-samples/debian/control [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Source: aero-mavlink-router-samples
+Section: embedded
+Priority: optional
```
Comment (avinash-palleti) [inline aero-mavlink-router-samples/debian/control]: Can you recheck Section and Priority fields.

File: aero-system/aero-system (hunk 1 above)
Comment (Rjasuja) [reply aero-system/aero-system]: Added dependency in aero-samples-apps

File: aero-mavlink-router-samples/debian/copyright [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: aero-mavlink-router-samples
+Source: https://github.com/01org/mavlink-router/tree/master/examples
+
+Files: *
+Copyright: 2017 Intel Corporation
+License: GPL-2
```
Comment (anselmolsm) [inline aero-mavlink-router-samples/debian/copyright]: MAVLink Router is Apache-2 - https://github.com/intel/mavlink-router/blob/master/LICENSE

**Review by anselmolsm — CHANGES_REQUESTED** [review]

File: aero-mavlink-router-samples/debian/copyright (hunk 3 above)
Comment (Rjasuja) [reply aero-mavlink-router-samples/debian/copyright]: Updated license

**Comment by anselmolsm** [comment]
Actually, this 'samples' package should leverage the existing mavlink-router package.
You can copy and adjust the .install file and extend aero-mavlink-router/debian/control. For reference, you can take one package that does something similar, e.g. [gtk3](http://archive.ubuntu.com/ubuntu/pool/main/g/gtk+3.0/gtk+3.0_3.18.9-1ubuntu3.debian.tar.xz)

**Comment by anselmolsm** [comment]
LGTM. Please squash the two commits.

**Pull request merged**